client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx

```diff
@@ -0,0 +1,185 @@
+				duration: translate( '%d minute', '%d minutes', { count: 2, args: [ 2 ] } ),
+				title: translate( 'Verify the email address for your domains' ),
+				description: translate(
+					'We need to check your contact information to make sure you can be reached. Please verify your details using the email we sent you, or your domain will stop working.'
+				),
+			};
+		case 'email_verified':
+			return {
+				duration: translate( '%d minute', '%d minutes', { count: 1, args: [ 1 ] } ),
+				title: translate( 'Confirm your email address' ),
+				description: translate(
+					'Please click the link in the email we sent to %(email)s.' +
+						'Typo in your email address? {{changeButton}}Change it here{{/changeButton}}.',
+					{
+						args: {
+							email: taskOptions?.userEmail,
+						},
+						components: {
+							br: <br />,
+							changeButton: <a href="/me/account" />,
+						},
+					}
+				),
+				//TODO: looks like there's some more complicated text states here
+				actionText: translate( 'Verify email' ),
+			};
+		case 'blogname_set':
+			return {
+				duration: translate( '%d minute', '%d minutes', { count: 1, args: [ 1 ] } ),
+				title: translate( 'Name your site' ),
+				description: translate(
+					'Give your new site a title to let people know what your site is about. ' +
+						'A good title introduces your brand and the primary topics of your site.'
+				),
+				actionText: translate( 'Name your site' ),
+				actionUrl: null,
+			};
+		case 'mobile_app_installed':
+			return {
+				duration: translate( '%d minute', '%d minutes', { count: 3, args: [ 3 ] } ),
+				title: translate( 'Get the WordPress app' ),
+				description: translate(
+					'Download the WordPress app to your mobile device to manage your site and follow your stats on the go.'
+				),
+				actionText: translate( 'Download mobile app' ),
+			};
+		case 'site_launched':
+			return {
+				duration: translate( '%d minute', '%d minutes', { count: 1, args: [ 1 ] } ),
+				title: translate( 'Launch your site' ),
+				description: translate(
+					"Your site is private and only visible to you. When you're ready, launch your site to make it public."
+				),
+				actionText: translate( 'Launch site' ),
+			};
+		case 'front_page_updated':
+			return {
+				duration: translate( '%d minute', '%d minutes', { count: 20, args: [ 20 ] } ),
+				title: translate( 'Update your Home page' ),
+				description: translate(
+					"We’ve created the basics, now it's time for you to update the images and text. Make a great first impression. Everything you do can be changed anytime."
+				),
+				actionText: translate( 'Create a menu' ),
+			};
+		case 'site_menu_updated':
+			return {
+				duration: translate( '%d minute', '%d minutes', { count: 10, args: [ 10 ] } ),
+				title: translate( 'Create a site menu' ),
+				description: translate(
+					'Building an effective navigation menu makes it easier for someone to find what they’re looking for and improve search engine rankings.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Building an effective navigation menu makes it easier for someone to find what they're looking for and improve search engine rankings.' )` *ES Score: 9*